extensions/blocks/instagram-gallery/instagram-gallery.php

```diff
@@ -57,8 +58,12 @@ function render_block( $attributes, $content ) {
 			'wp-block-jetpack-instagram-gallery__grid-columns-' . $columns,
 		)
 	);
-	$grid_style   = 'grid-gap: ' . $spacing . 'px;';
-	$photo_style  = 'padding: ' . $spacing . 'px;';
+	if ( $is_stacked_on_mobile ) {
```

Comment:
Maybe this could be simplified a bit?

```diff
diff --git a/extensions/blocks/instagram-gallery/instagram-gallery.php b/extensions/blocks/instagram-gallery/instagram-gallery.php
index c701e2ac1..894c17069 100644
--- a/extensions/blocks/instagram-gallery/instagram-gallery.php
+++ b/extensions/blocks/instagram-gallery/instagram-gallery.php
@@ -56,11 +56,9 @@ function render_block( $attributes, $content ) {
 		array(
 			'wp-block-jetpack-instagram-gallery__grid',
 			'wp-block-jetpack-instagram-gallery__grid-columns-' . $columns,
+			( $is_stacked_on_mobile ? 'is-stacked-on-mobile' : null ),
 		)
 	);
-	if ( $is_stacked_on_mobile ) {
-		$grid_classes .= ' is-stacked-on-mobile';
-	}
 
 	$grid_style  = 'grid-gap: ' . $spacing . 'px;';
 	$photo_style = 'padding: ' . $spacing . 'px;';

```